app/src/main/res/layout/item_status.xml

```diff
@@ -103,6 +103,40 @@
         app:layout_constraintTop_toTopOf="@id/status_display_name"
         tools:text="13:37" />
 
+    <Button
+        android:id="@+id/status_button_untranslate"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/action_show_original"
+        app:layout_constraintTop_toBottomOf="@id/status_username"
+        style="@style/TuskyButton.TextButton"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginEnd="10dp"
+        android:visibility="gone"
+        tools:visibility="visible"
+        android:minHeight="0dp"/>
+
+    <TextView
+        android:id="@+id/status_translation_status"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        style="@style/TextSizeSmall"
+        tools:text="Translated from Lang by Service"
+        app:layout_constraintStart_toStartOf="@id/status_display_name"
+        app:layout_constraintEnd_toStartOf="@id/status_button_untranslate"
+        app:layout_constraintBaseline_toBaselineOf="@id/status_button_untranslate"
+        android:layout_marginEnd="4dp"
+        android:maxLines="4"
+        android:visibility="gone"
+        tools:visibility="visible" />
```

Comment:
I'd add a `lineSpacingMultiplier="1.1"`, align the button top and bottom to the text, and add 4dp top margin.
Like this:
<img src="https://github.com/tuskyapp/Tusky/assets/10157047/492f6f6d-103e-42c6-a0c2-aa218ee41158" width="300" />

Reply:
I'm a bit confused, currently the first line is aligned with the button text, would you rather center it relative to the text? To me both looks good, I will try your idea with centering!

Reply:
padding between them, right?

Reply:
![Screenshot from 2024-03-05 20-52-33](https://github.com/tuskyapp/Tusky/assets/3099142/33af6e23-177f-4f9f-970f-041c2c2d3bcb)

idk how to both center the button to the text but also not make button overlap above?

Reply:
> padding between them, right?

4dp additional padding between username and text, but of course there should also be a bit of padding between text & button.

> idk how to both center the button to the text but also not make button overlap above?

A yes when the text is only one line it can overlap 🤔 
Some possibilities:
- another barrier
- force the text to be at least 2 lines
- leave it top aligned
- 
I don't care much.